default.nix

```diff
@@ -1 +1,7 @@
-let pkgs = import ./nix { }; in pkgs.packages
+let
+  sources = import ./nix/sources.nix;
+  pkgs = sources.nixpkgs;
+  system = builtins.currentSystem;
+  _pkgs = import ./nix { inherit sources system; nixpkgs = pkgs; };
+in
+_pkgs.packages
```

Comment:
Why duplicate the wiring? These are all default arguments.

Reply:
That is.... a good question and the answer is that i just amended my commit. Thanks @roberth